### PR TITLE
Discourse link

### PR DESCRIPTION
**Pull request by felixbbertrand**
change dead discourse link  https://metagame.trydiscourse.com/  to  https://forum.metagame.wtf/

### CODE REVIEW

**Comment by felixbbertrand** [comment]
@peth-yursick

File: docs/how-does-it-work/xp.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: 🧬 XP & Seeds 🌱
 
 ## **📊 How does XP work?**
 
-Whether it’s completing random quests, sharing things on [Discord](https://discord.gg/ZqdPP9b), writing useful stuff on [Discourse](https://metagame.trydiscourse.com/), or laying down code on [GitHub](https://github.com/metafam/) - your XP is automagically being accrued.
+Whether it’s completing random quests, sharing things on [Discord](https://discord.gg/ZqdPP9b), writing useful stuff on [Discourse](https://forum.metagame.wtf/), or laying down code on [GitHub](https://github.com/metafam/) - your XP is automagically being accrued.
```
Comment (luxumbra) [inline docs/how-does-it-work/xp.md:11]: Thanks for this. It actually made me realise that after 4 months of MG I'd not set my Discourse username. :laughing:

**Review by luxumbra — APPROVED** [review]
Well spotted. :pray:

File: docs/how-does-it-work/xp.md (hunk 1 above)
Comment (felixbbertrand) [reply docs/how-does-it-work/xp.md:11]: @luxumbra  😆

**Pull request merged**